components/footer.js

```diff
@@ -31,7 +32,7 @@ export default () => (
             </a>
           </li>
           <li>
-            <a href="https://www.meetup.com/coderplex/">m</a>
+            <a href="https://www.twitter.com/coderplex/"><FaTwitter size={20}</a>
```

Comment:
Hi, @loangelak
You forgot to close `FaTwitter` tag . It should be `<FaTwitter size={20}/>`